src/accordion.ts

```diff
@@ -97,8 +97,14 @@ export default class Accordion extends LitElement {
                 opacity: [0, 1],
               },
               {
-                duration: 150,
-                easing: 'ease-in',
+                duration: Number(
+                  getComputedStyle(this.#defaultSlotElementRef.value)
+                    .getPropertyValue('--private-open-duration')
+                    .replace('ms', ''),
```

Comment:
Our durations use `ms`, so we need to lob it off. Otherwise, you see:

```bash
accordion.ts?t=1752844224704:69 Uncaught (in promise) TypeError: Failed to execute 'animate' on 'Element': duration must be non-negative or auto
    at accordion.ts?t=1752844224704:69:60
```

and it doesn't animate.